src/server/script_mgr.cc

```diff
@@ -38,6 +38,9 @@ ABSL_FLAG(bool, lua_allow_undeclared_auto_correct, false,
           "access undeclared keys, automaticaly set the script flag to be able to run with "
           "undeclared key.");
 
+ABSL_FLAG(std::vector<std::string>, lua_undeclared_keys_shas, {},
```

Comment:
Should we default this flag's value to `351130589c64523cb98978dc32c64173a31244f3` ?

Reply:
Why not? it is strictly better to do so no?

Reply:
I agree that it sounds better. There are 2 caveats I can think of:
1. Updating / downgrading Sidekiq version could cause this to stop working, confusing users
2. There is a very small runtime penalty of accessing a variable via a mutex and then copying vector+string, which will affect non-sidekiq users. However this will only happen on `EVAL` or `LOAD` (not `EVALSHA`) and should not be on the hot path, so I guess it's not a big deal...
Anyway, I'll add that.

Reply:
+1 to keep sha in the flag.